src/app.ts

```diff
@@ -47,7 +53,9 @@ nunjucksEnv.addGlobal("OE_CONFIGS", config);
 nunjucksEnv.addGlobal("ERROR_MESSAGES", ErrorMessages);
 nunjucksEnv.addGlobal("COUNTRY_FILTER", countryFilter );
 nunjucksEnv.addGlobal("CREATE_CHANGE_LINK", createChangeLinkConfig);
+nunjucksEnv.addGlobal("CREATE_CHANGE_LINK_WITH_IDs", createChangeLinkWithIds);
```

Comment:
All constant letters should be capitalised.

Reply:
Ok, will changes this.

Was trying to make it obvious to the user that this should read as "IDs" and not "I.D.S." 😄